_inc/client/components/admin-notices/index.jsx

```diff
@@ -7,11 +7,16 @@ const AdminNotices = React.createClass( {
 		let $vpNotice = jQuery( '.vp-notice' );
 		if ( $vpNotice.length > 0 ) {
 			$vpNotice.each( function () {
+				if ( jQuery( this ).children( '.vp-message' ).children( 'p' ).html().includes( 'VP08' ) ) {
+					jQuery( this ).children( '.vp-message' ).children( 'p' ).html( 'Your VaultPress subscription is no longer active. Please confirm the status of your account. <a href="https://dashboard.vaultpress.com/account/">View account</a>' );
```

Comment:
> Perhaps we should just translate this particular string in Jetpack?

Maybe that's the simplest solution? That would solve our problem here. We can update the string in VaultPress as well later on.